README.md

```diff
@@ -1,13 +1,21 @@
 # Project Happy Thoughts API
 
-Replace this readme with your own information about your project.
-
-Start by briefly describing the assignment in a sentence or two. Keep it short and to the point.
+Backend part for the Happy Projects frontend project: https://github.com/ankimelin/project-happy-thoughts, including a database (MongoDB) and an API on top of that (Mongoose).
```

Comment:
It is always inspiring to do a code review on you Annika. You always challenge yourself. Good work!